spec/system/sign_in_redirect_spec.rb

```diff
@@ -0,0 +1,118 @@
+require "rails_helper"
+
+RSpec.describe "Sign in redirect", service: :placements, type: :system do
```

Comment:
This is _potentially_ different per service. If we were to change this behaviour in the Claims app, no test would currently cover it.

Can we duplicate this test and put these in according directories please?

Reply:
This feels like a change that is outside the scope of this PR as it would make this test _different_ to the existing system specs, it is also unlikely to add any value to the integrity of the codebase given that the logic tested here will not differ between our services.

If this is something you feel strongly about, we can look into refactoring all of the tests to be specific to each service, but we should avoid doing so unless we can prove that it's adding value to the codebase.

Reply:
If I add the following line into the code, would it break any specs?

```ruby
class ApplicationController < ActionController::Base
   def requested_path
    return if [sign_in_path, sign_out_path].include?(session["requested_path"])

    return session["requested_path"] if service == :claims

    @requested_path ||= session.delete("requested_path")
  end
end
```